### PR TITLE
[strings] Trails translations

### PR DESCRIPTION
**Pull request by biodranik**
Please check these translations: https://github.com/organicmaps/organicmaps/commit/eb95b8c5c36f630e1562492e7f2cb143d12ff903

### CODE REVIEW

**Review by NeatNit — CHANGES_REQUESTED** [review]
some changes needed in Hebrew

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -12850,17 +12855,93 @@
...
+    id = Jejak yang sulit atau kurang terlihat
     it = Sentiero difficile o poco tracciato
+    ja = 困難な、または見通しの悪いトレイル
+    ko = 어렵거나 잘 보이지 않는 트레일
+    lt = Sudėtingas arba blogai matomas takas
+    mr = अवघड किंवा खराब दृश्यमान पायवाट
+    nb = Vanskelig eller dårlig synlig sti
+    nl = Moeilijk of slecht zichtbaar pad
+    pl = Trudny lub słabo widoczny szlak
+    pt = Percurso difícil ou pouco visível
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:12886]:
```suggestion
    pt = Trilho difícil ou pouco visível
```

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -12850,17 +12855,93 @@
...
+    id = Jejak yang sangat sulit atau tidak dapat dibedakan
     it = Sentiero estremamente difficile o molto poco tracciato
+    ja = 非常に難しい、または区別がつかないトレイル
+    ko = 매우 어렵거나 구별하기 어려운 흔적
+    lt = Labai sudėtingas arba neišsiskiriantis takas
+    mr = अतिशय अवघड किंवा वेगळे न करता येणारी पायवाट
+    nb = Svært vanskelig eller umulig å skille ut stien
+    nl = Zeer moeilijk of niet te onderscheiden spoor
+    pl = Bardzo trudny lub nierozróżnialny szlak
+    pt = Percurso muito difícil ou indistinguível
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:12932]:
```suggestion
    pt = Trilho muito difícil ou indistinguível
```

**Review by matheusgomesms — COMMENTED** [review]
PT and PT-BR review

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -12850,17 +12855,93 @@
...
+    uk = Складна або погано видима стежка
+    vi = Đường đi khó hoặc khó nhìn thấy
+    zh-Hans = 难走或难见的小路
+    zh-Hant = 困難或不明顯的小路
 
   [type.highway.path.expert]
     comment = Hiking trail tagged as sac_scale=alpine_hiking (4+ of 6) or trail_visibility=horrible or more extreme.
     ref = type.highway.path
-    en = Expert or Indiscernible Trail
+    en = Expert or indistinguishable Trail
```
Comment (map-per) [inline data/strings/types_strings.txt:12903]:
```suggestion
    en = Expert or pathless Trail
```

I'm not a native speaker, but does "indistinguishable" really make sense her? "Indiscernible" or "pathless" are better IMO.

Comment (RedAuburn) [reply data/strings/types_strings.txt:12903]: pathless sounds strange. Indiscernible makes most sense IMO, but indistinguishable works too

**Review by d4f5409d — APPROVED** [review]
Hu LGTM

**Comment by jimcarst** [comment]
nl is ok

File: data/strings/types_strings.txt (hunk 3 above)
Comment (map-per) [reply data/strings/types_strings.txt:12903]: Pathless comes from the [Wiki](https://wiki.openstreetmap.org/wiki/Key:trail_visibility). But yes, a pathless path is to some degree an oxymoron.

**Review by Arthur-GYT — APPROVED** [review]
LGTM for French

File: data/strings/types_strings.txt [hunk 4]
```diff
@@ -12893,19 +12897,93 @@
...
+    hu = Nagyon nehéz vagy megkülönböztethetetlen nyomvonal
+    id = Jejak yang sangat sulit atau tidak dapat dibedakם)
     it = Sentiero estremamente difficile o molto poco tracciato
+    ja = 非常に難しい、または区別がつかないトレイル
+    ko = 매우 어렵거나 구별하기 어려운 흔적
+    lt = Labai sudėtingas arba neišsiskiriantis takas
+    mr = अतिशय अवघड किंवा वेगळे न करता येणारी पायवाट
+    nb = Svært vanskelig eller umulig å skille ut stien
+    nl = Zeer moeilijk of niet te onderscheiden spoor
+    pl = Bardzo trudny lub nierozróżnialny szlak
```
Comment (mpawelski) [inline data/strings/types_strings.txt:12973]: "Bardzo trudny lub niedostrzegalny szlak" sounds better

Comment (biodranik) [reply data/strings/types_strings.txt]: Thanks! Fixed.

**Comment by biodranik** [comment]
@vng PTAL

**Pull request merged**